packages/orbit-components/src/common/tailwind/wrap.ts

```diff
@@ -0,0 +1,33 @@
+import { QUERIES } from "../../utils/mediaQuery/consts";
+
+enum WRAP {
+  WRAP = "wrap",
+  NO_WRAP = "nowrap",
+}
+
+export const wrapClasses: {
+  [K in QUERIES | WRAP]: K extends QUERIES ? Record<WRAP, string> : string;
+} = {
+  [WRAP.WRAP]: "flex-wrap",
+  [WRAP.NO_WRAP]: "flex-nowrap",
+  [QUERIES.LARGEDESKTOP]: {
```

Comment:
Can you define queries as top->down approach? I saw it before already and I found it inconsistent, let's define them top->down from mediuMobile to largeDesktop, for better readability

Reply:
Agree, just if it can be fixed during this change -> it will be also good :)